### PR TITLE
Export Issue Comments to Rockset

### PR DESCRIPTION
**Pull request by ZainRizvi**
This will enable us to track all commands made to pytorchmergebot, letting us more easily track reverts, force commits, etc.

We'll separately require more parameters to be sent to pytorchmergebot for those commands to get more details logged

### CODE REVIEW

**Review by huydhn — APPROVED** [review]

**Comment by huydhn** [comment]
Just to drop a context here on where to find the table and Rockset collections (created manually):

* The DynamoDB table is [torchci-issue-comment](https://us-east-1.console.aws.amazon.com/dynamodbv2/home?region=us-east-1#table?initialTagKey=&name=torchci-issue-comment&tab=overview)
* The Rockset collection name is [issue_comment](https://console.rockset.com/collections/details/commons.issue_comment?tab=summary)

File: torchci/lib/bot/webhookToDynamo.ts [hunk 1]
```diff
@@ -64,6 +64,22 @@ async function handleIssues(event: WebhookEvent<"issues">) {
...
+  event: WebhookEvent<"issue_comment">
+) {
+  const key_prefix = event.payload.repository.full_name;
+  const client = getDynamoClient();
+
+  await client.put({
+    TableName: "torchci-issue-comment",
+    Item: {
+      // Still need uuidv4() at the end to keep the key unique when a comment is edited, since the comment.id doesn't change
+      dynamoKey: `${key_prefix}/${event.payload.issue.number}/${event.payload.comment.id}/${uuidv4()}`,
```
[the remark this answers is not in the document]
Comment (zengk95) [reply torchci/lib/bot/webhookToDynamo.ts:77]: For example, we do make the key just the issueID in L61 because we don't want to deal with this.

**Review by zengk95 — APPROVED** [review]

File: torchci/lib/bot/webhookToDynamo.ts [hunk 2]
```diff
@@ -64,6 +64,22 @@ async function handleIssues(event: WebhookEvent<"issues">) {
...
+async function handleIssueComment(
+  event: WebhookEvent<"issue_comment">
+) {
+  const key_prefix = event.payload.repository.full_name;
+  const client = getDynamoClient();
+
+  await client.put({
+    TableName: "torchci-issue-comment",
+    Item: {
+      // Still need uuidv4() at the end to keep the key unique when a comment is edited, since the comment.id doesn't change
```
Comment (ZainRizvi) [inline torchci/lib/bot/webhookToDynamo.ts:76]:
```suggestion
```

File: torchci/lib/bot/webhookToDynamo.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ZainRizvi) [reply torchci/lib/bot/webhookToDynamo.ts]: Ah, didn't realize this was an upsert command. That's even better!

**Pull request merged**